pages/common/forever.md

```diff
@@ -0,0 +1,15 @@
+# forever - nodejs tool
```

Comment:
The generally prefer the title of a page to be the same as the name of command.

Reply:
Let me know if you're asking me to change it to "forever"